src/config.c

```diff
@@ -100,6 +100,37 @@ struct Config *getCommonConfig(void) {
     return &commonConfig;
 }
 
+// Reloads the configuration file and removes interfaces which were removed from config.
+void reloadConfig() {
```

Comment:
igmpproxy is C not C++. Function with *no* arguments is declared in C as `void reloadConfig(void)`.

`void reloadConfig()` means function with unspecified arguments. It may be zero, ten, whatever.

Reply:
fixed

Reply:
fixed